inference-engine/src/low_precision_transformations/src/network_helper.cpp

```diff
@@ -800,6 +800,23 @@ FakeQuantizeDequantization NetworkHelper::getDequantization(const std::shared_pt
     return FakeQuantizeDequantization(dataNode, convert, subtract, multiply);
 }
 
+void NetworkHelper::normalizeDequantization(FakeQuantizeDequantization& dequantization) {
+    if (dequantization.multiply != nullptr && as_type_ptr<ngraph::opset1::Constant>(dequantization.multiply->get_input_node_shared_ptr(0))) {
+        std::shared_ptr<Node> leftParent = dequantization.multiply->get_input_node_shared_ptr(0);
+        std::shared_ptr<Node> rightParent = dequantization.multiply->get_input_node_shared_ptr(1);
+        std::shared_ptr<opset1::Multiply> normalized_multiply = as_type_ptr<opset1::Multiply>(
+                dequantization.multiply->clone_with_new_inputs({rightParent, leftParent}));
+        *dequantization.multiply = *normalized_multiply;
```

Comment:
`dequantization.multiply = normalized_multiply;` should be enough, but I think returning new dequantization is more clear

Reply:
This will change only the class field, but not the graph.

Reply:
I think returning new dequantization instance is more clear, don't update input argument